### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by Kossi-stack**
This is my past project during the microverse trials, I am requesting a review.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+            <img src="dr-borlaug.jpg" alt="Dr.Borlaug" id="image" class="center">
+
+            <div id="img-caption">
+                <p>Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields
+                    - part of his life-long war on hunger. </p>
+
+            </div>
+        </div>
+    </div>
+
```
Comment (Kossi-stack) [inline index.html:30]: Please do not use spaces, or indentations without a reason. For readability, add two spaces of indentation.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  }
+
+  img {
+    max-width: 100%;
+    height: auto;
+    margin-left: auto;
+    margin-right: auto;
+    width: 100%;
+  }
+}
```
Comment (Kossi-stack) [inline style.css:86]: For CSS spaces and indentations is being well structured for readability.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+    <footer>
+        <h3>
+            If you have time, you should read more about this incredible human being on his <a
+                href="https://en.wikipedia.org/wiki/Norman_Borlaug" id="tribute-link" target="_blank"
+                rel="noopener noreferrer">Wikipedia entry.</a>
+        </h3>
+    </footer>
+
```
Comment (Kossi-stack) [inline index.html:126]: Please do not add blank lines or spaces without a reason. Add blank lines to separate large or logical code blocks.

**Review by Kossi-stack — COMMENTED** [review]
Required Changes ♻️ 
 
Please check my comments below and make the required changes.

Submit for a re-review when you are done.

Thanks 😁

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @Kossi-stack 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

### **Highlights**

- Politely asked the request and why you requested it 💯
- Using inline comments 💯

## **OPTIONAL**

- [ ] It'd be better if you added some emojis to your comments. It helps to make it friendly and kind 🥰 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**